app.py

```diff
@@ -242,6 +287,26 @@ def __add_write_node_commands(self, context=None):
                     context=context,
                 )
             )
-            
-            
 
+        # Show the convert actions in the Menu if configured to do so
+        if self.get_setting("show_convert_actions"):
+
+            convert_to_write_nodes_action = lambda :self.convert_to_write_nodes(show_warning=True)
+            convert_from_write_nodes_action = lambda: self.convert_from_write_nodes(show_warning=True)
+
+            self.engine.register_command(
+                "Convert All SG Write Nodes to Write Nodes...",
```

Comment:
nitpick: Given the dialogs, i think we can now say "Convert SG Write Nodes to Write Nodes..."